summary.tex

```diff
@@ -1,16 +1,19 @@
 \section{Summary}
 
-Rubin Observatory is putting in place a dedicated {\it Early Science Program} to ensure high-impact science during the first year of operation of the \lsst.
-This Early Science Program is motivated by the fact that in the current baseline there will be no science-ready data products released before \drone. 
+Rubin Observatory is putting in place a dedicated {\it Early Science Program} to ensure high-impact science prior to the release of the first year of \lsst data.
+The Rubin Operations team is carrying out a series of ``Data Previews,'' in which LSST pre-cursor data products are prepared, released to the LSST data rights community, and supported.
+The goals of the Data Previews are to 1) enable the Operations teams to develop operational capability prior to the start of the LSST, and 2) support the members of the LSST science community as they develops their LSST analyses.
+The first Data Preview, DP0, has already been released: it contains Rubin-processed image and catalog data products derived from simulated LSST images that were generated by the LSST Dark Energy Science Collaboration as the DC2 Virtual Sky Survey \citep{2021ApJS..253...31L}.
+Subsequent Data Previews DP1 and DP2 will contain LSST-like data products generated from the prompt processing (PP) and data release processing (DRP) of the commissioning data taken with ComCam and LSST Cam, respectively.
```

Comment:
```suggestion
Subsequent Data Previews DP1 and DP2 will contain LSST-like data products generated from processing the commissioning data taken with ComCam and LSST Cam, respectively.
```

I'm a little unsure whether we call AP products "part of" DP1/2.  For DP1 the "canned" alerts are likely to be released at about the same time as the DP1 DRP processing, but with LSSTCam we hope the alerts might come ahead of the DP1 DRP release?  The proposed phrasing just sidesteps the confusion--we go into gory detail below.

Reply:
Agree - let's keep it non-specific at this time.